client/src/reducers/datasets.js

```diff
@@ -19,6 +19,37 @@ function saveDatasetSettings(state, dataset) {
   });
 }
 
+function importDatasetPending(state, { importId, name }) {
+  const now = Date.now();
```

Comment:
Perhaps we should grab the date in the action creator instead, to keep the reducers free of calls to non-pure functions? Same goes for line 36